readthedocs/restapi/views/integrations.py

```diff
@@ -264,6 +320,19 @@ class APIWebhookView(WebhookMixin, APIView):
         {
             "branches": ["master"]
         }
+
+    Or the following JSON::
+
+        {
+            "branches": [{
+                "name": "branch-name",
+                "last_commit": {
+                    "id": "sha",
+                    "message": "Update README.md"
+                }
+            }]
+        }
+
```

Comment:
This is what I proposed for the general webhook, maintaining the backwards compatibility.